www-cronrobot/Gemfile

```diff
@@ -64,7 +64,7 @@ group :development do
 end
 
 group :production do
-  gem 'pg'
+ gem 'pg'
 end
 
 group :test do
```

Comment:
the private key field should be a text-field

![image](https://user-images.githubusercontent.com/8007722/132594127-f57f942b-4e03-4603-aa63-efb6b6cd83f2.png)

a private key looks like the following, so you can test with a 20 lines text approximately:

https://www.google.com/url?sa=i&url=https%3A%2F%2Fdocs.rackspace.com%2Fsupport%2Fhow-to%2Flogging-in-with-an-ssh-private-key-on-linuxmac&psig=AOvVaw1zZQEpL1RMtX_iPeSje2YF&ust=1631226587271000&source=images&cd=vfe&ved=0CAsQjRxqFwoTCOCo39y28PICFQAAAAAdAAAAABAD

Reply:
I am unable to fix this because I can't access this page to test it out 😞

Reply:
Fixed 👍